### ISSUE TITLE
WCAG 4.1.2: Ensures ARIA attributes are allowed for an element's role (.is--copyright)

### ISSUE THREAD

**Issue by davidsneighbour** [davidsneighbour/kollitsch.dev#569]
#### Issue
`Ensures ARIA attributes are allowed for an element's role` ([aria-allowed-attr](https://accessibilityinsights.io/info-examples/web/aria-allowed-attr))
#### Target application
[Reusable Gohugo Forms via Configuration ↭ KOLLITSCH.dev*](https://behemoth:1313/blog/2023/reusable-gohugo-forms-via-configuration/)
#### Element path
.is--copyright
#### Snippet
`<div class="is--copyright" data-bs-toggle="tooltip" data-bs-placement="top" data-bs-title="Carefully implemented forms" aria-label="Carefully implemented forms" data-bs-original-title="Carefully implemented forms">`
#### How to fix
    Fix all of the following:
      aria-label attribute cannot be used on a div with no valid role attribute.
#### Environment
Chrome version 118.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.40.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
